declarations/plugins/DotenvPlugin.d.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * This file was automatically generated.
+ * DO NOT MODIFY BY HAND.
+ * Run `yarn fix:special` to update
+ */
+
+export interface DotenvPluginOptions {
+	/**
+	 * Whether to allow empty strings in safe mode. If false, will throw an error if any env variables are empty (but only if safe mode is enabled).
+	 */
+	allowEmptyValues?: boolean;
```

Comment:
Should we set it `true` by default? Can you look at nextjs/vite, how they work here?